src/Timeline.js

```diff
@@ -74,25 +75,28 @@ export default class Timeline extends Axis {
               ? event.selection 
               : [event.selection[0] + buttonDomain, event.selection[1] - buttonDomain].sort((a, b) => a - b);
 
-      const domain = (this._brushing ? selection
-        : this._buttonBehaviorCurrent === "buttons" 
-          ? [event.sourceEvent.offsetX, event.sourceEvent.offsetX] : [event.selection[0], event.selection[0]])
-        .map(this._d3Scale.invert)
-        .map(Number);
+      const domain = this._buttonBehaviorCurrent === "ticks"
+        ? (this._brushing 
+          ? event.selection 
+          : [event.sourceEvent.offsetX, event.sourceEvent.offsetX]).map(this._d3Scale.invert).map(Number) 
+        : (this._brushing 
+          ? selection
+          : [event.sourceEvent.offsetX, event.sourceEvent.offsetX]).map(Number);
```

Comment:
can we move these `domain` and `ticks` variables into the updateDomain function as well?